crates/ruma-events/src/forwarded_room_key.rs

```diff
@@ -43,6 +43,10 @@ pub struct ToDeviceForwardedRoomKeyEventContent {
     /// key is forwarded from A to B to C, this field is empty between A and B, and contains
     /// A's Curve25519 key between B and C.
     pub forwarding_curve25519_key_chain: Vec<String>,
+
+    /// Used to mark key if allowed for shared history
+    #[serde(default, rename = "org.matrix.msc3061.shared_history")]
```

Comment:
We probably also want to use `skip_serializing_if(ruma_common::serde::is_default)`